### ISSUE TITLE
Global variables bugs

### ISSUE THREAD

**Issue by JimB40** [EdgeTX/edgetx#322]
1. GV name disappear in edit mode if GV was active then deactivated (test with Toggle switch)
2. Empty leftover after Clear command
![122798111-801efb80-d2c0-11eb-8191-df69d8a42bcb](https://user-images.githubusercontent.com/46420768/123134043-aa9fbe80-d450-11eb-8bc3-c0f1b8927833.png)

**General information**
 - EdgeTX 68cb9c2
 - RMTX16S

**Comment by pfeerick** [comment]
GV (Global Value) or GF (Global Function)? If it's the later, I'm still on the latest nightly (so doesn't include that commit, but I don't think that matters) , and can reproduce that - i.e. GF3 for me is for screenshot of SHdn and I initially have GF3 in white in the header, yellow while active, and then it vanishes. 

Not sure there's anything wrong with that second point... clear is supposed to clear, not delete. If you delete instead of clear, it removes the row. If you clear, it clears the values, but leaves the row. What is a bug though is (at the global functions screen) if you clear instead of delete (as you have the choice of both initially), the context menu that appears for that row after clearing  only has Clear as an option. i.e. you've lost the option for delete, which in this case is the only valid option since it has already been cleared.

**Comment by JimB40** [comment]
After ‘clear’ GF3 line should look like GF4 or GF5 on screenshot. Values are cleared It’s just UI display bug. If you change tab and go back it look correctly

**Comment by pfeerick** [comment]
Latest nightly, no. 1 is till present. 

Ah, I'm with you now... to resolve no. 2, instead of this when clearing... 

![image](https://user-images.githubusercontent.com/5500713/123197297-ac1fc400-d4ee-11eb-8b61-e2677360c8eb.png)

... you want it to look like this... 

![image](https://user-images.githubusercontent.com/5500713/123197316-b3df6880-d4ee-11eb-86c4-aea61f5fcfa7.png)

... which is consistent which how it looks if you revisit the screen later. Yeah, oopsie!

**Comment by pfeerick** [comment]
#325 Addresses point 1 of this issue, so just point 2 needs doing ;)

**Comment by Jhawk736** [comment]
Hardware is Jumper T16 using EdgeTx 2.6
Radio Global functions don't work.
Instead, special function which refers to each model act as global function.
Was setting S2 as volume control, its works via special function.
Is global function now disabled in the version ?

**Comment by pfeerick** [comment]
No, global function for volume (and brightness) still work - perhaps you have not configured or enabled the global function - It should be configured like so... make sure the swith is set to ON (unless you use a switch to enable and disable pot volume control) and the function is enabled. 

![image](https://user-images.githubusercontent.com/5500713/155626992-4fe3b35b-2f7b-4571-903f-d2819f0eb15b.png)

**Comment by Jhawk736** [comment]
Yes, exactly the same settings. I notice that I cannot enable it. When clicking the enable option, the GF1 letters should turn yellow.
Mine does not, but Special functions do turn yellow when enabling.
Any idea? Fw is edgetx-t16, thanks.

**Comment by Jhawk736** [comment]
Further digging, found that Global function works in the 2nd model. 
Going back to the 1st model, Global function settings remains, but the enable button still don't work.
I have to keep screen and volume in Special function in order for Global function to work in other models.
It's weird but it's good to go.
